smartsim/settings/palsSettings.py

```diff
@@ -185,6 +185,16 @@ def set_walltime(self, walltime):
         """
         logger.warning("set_walltime not supported under PALS")
 
+    def set_gpu_affinity_script(self, affinity, *args):
```

Comment:
So, here the idea would be to let the user add more than one file?
Also, I think you will need to add typehints, so possibly the declaration will be:
```python
def set_gpu_affinity_script(self, affinity: str, *args: str) -> None: